color-picker/src/commonMain/kotlin/com/godaddy/android/colorpicker/harmony/HarmonyColorPicker.kt

```diff
@@ -73,11 +73,17 @@ fun HarmonyColorPicker(
     )
 }
 
+/**
+ *
+ * Will show a brightness bar if [fixedBrightness] is null
+ * otherwise all colors are given the provided brightness value
+ */
 @Composable
 fun HarmonyColorPicker(
     modifier: Modifier = Modifier,
     harmonyMode: ColorHarmonyMode,
     color: HsvColor,
+    fixedBrightness: Float? = null,
```

Comment:
Please, to keep consistency with ClassicColorPicker, please rename it to `showBrightnessBar`.

if a user wants a specific brightness, it should set showBrightnessBar to false and set the brightness in the provided color.

Reply:
@ffgiraldez thanks for pointing that out. I didn't notice ClassicColorPicker had that.   Updated now